### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by HatEmU**
Note: "Latter" refers to the pronoun that comes later or near the end and "Former" refers first pronoun.

### CODE REVIEW

File: flutter_dropzone/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Stack(
 ## Using it in cross-platform apps
 
 It's a federated plugin, meaning that it will compile in cross platform apps that contain both Android/iOS and Web code.
-It will *not* function on the latter, the view will simply return an error text instead of a drop zone. Use `if (kIsWeb)` from
+It will *not* function on the former, the view will simply return an error text instead of a drop zone. Use `if (kIsWeb)` from
 `import 'package:flutter/foundation.dart'` to only use it in Flutter Web. Still, the same app will still compile to
```
Comment (deakjahn) [inline flutter_dropzone/README.md:37]: Fair enough. Probably I changed the wording of the first sentence and forgot to adapt the second. Or whatever. :-)

File: flutter_dropzone/README.md [hunk 2]
```diff
@@ -62,4 +62,4 @@ You can get a temporary URL using:
 *  `Future<bool> releaseFileUrl(String fileUrl);`
 
 but this will only be valid for the session. It's a regular URL, so you can use it to display the image the same way like loading
-from a regular web URL. Release it when you're done with the image.
\ No newline at end of file
+from a regular web URL. Release it when you're done with the image.
```
Comment (deakjahn) [inline flutter_dropzone/README.md:65]: Well, what is the difference exactly? I can't see...

Comment (deakjahn) [reply flutter_dropzone/README.md:65]: Oh. Plus one newline. Probably not terribly important. :-)

Comment (HatEmU) [reply flutter_dropzone/README.md:65]: oops no I didn't mean to do that... I was using github.com to make this pull request I guess it created a new line on its own

**Pull request merged**